cg/cli/base.py

```diff
@@ -65,22 +65,20 @@ def base(
 @base.command()
 @click.option("--reset", is_flag=True, help="reset database before setting up tables")
 @click.option("--force", is_flag=True, help="bypass manual confirmations")
-@click.pass_obj
-def init(context: CGConfig, reset: bool, force: bool):
+def init(reset: bool, force: bool):
     """Setup the database."""
-    status_db: Store = context.status_db
-    existing_tables = status_db.engine.table_names()
+    existing_tables = get_tables()
     if force or reset:
         if existing_tables and not force:
             message = f"Delete existing tables? [{', '.join(existing_tables)}]"
             click.confirm(click.style(message, fg="yellow"), abort=True)
-        status_db.drop_all()
+        drop_all_tables()
     elif existing_tables:
         LOG.error("Database already exists, use '--reset'")
         raise click.Abort
 
-    status_db.create_all()
-    LOG.info("Success! New tables: %s", ", ".join(status_db.engine.table_names()))
+    create_all_tables()
+    LOG.info("Success! New tables: %s", ", ".join(get_tables()))
```

Comment:
```suggestion
    LOG.info("Success! New tables: %s", ", ".join(get_tables()))
```
can we make this an f string? or out of scope?